topics/server.md

```diff
@@ -0,0 +1,86 @@
+---
+title: "Valkey Server"
+linkTitle: "Valkey Server"
+weight: 1
+description: >
+    Manual for valkey-server, the Valkey server program
+---
+
+## Usage
+
+**`valkey-server`** [ _/path/to/valkey.conf_ ] [ _OPTIONS_ ] [**`-`**]\
+**`valkey-server`** **`-`**\
```

Comment:
```suggestion
**`valkey-server`** [ _/path/to/valkey.conf_ ] [ _OPTIONS_ ] [**`-`**]\
```

Isn't the second example just a sub-example of the first one?

Reply:
Apparently. Remove it?

I just took this from `valkey-server --help`.